deploy.sh

```diff
@@ -38,6 +38,13 @@ if [[ -z "$ASSETS_DIR" ]]; then
 fi
 echo "ℹ︎ ASSETS_DIR is $ASSETS_DIR"
 
+if [[ -z "$BUILD_DIR" ]]; then
+	BUILD_DIR=false
+elif [[ $BUILD_DIR != /* ]]; then 
+	BUILD_DIR="${GITHUB_WORKSPACE%/}/${BUILD_DIR}"
+fi
+echo "ℹ︎ BUILD_DIR is $BUILD_DIR"
```

Comment:
```suggestion
if [[ -z "$BUILD_DIR" ]]; then
	BUILD_DIR=false
	echo "ℹ︎ Building from repository root directory"
elif [[ $BUILD_DIR != /* ]]; then 
	BUILD_DIR="${GITHUB_WORKSPACE%/}/${BUILD_DIR}"
	echo "ℹ︎ BUILD_DIR is $BUILD_DIR"
fi
```

Avoids `BUILD_DIR is false` message for something a little clearer

Reply:
I address this in 43e313a, I decided to not echo anything if `BUILD_DIR` is `false` because it's the current behavior.